pkg/migrate/migrate.go

```diff
@@ -41,13 +48,30 @@ var isDestScLocalVolumeProvisioner bool
 
 // Options is the set of options that should be provided to Migrate
 type Options struct {
-	SourceSCName         string
-	DestSCName           string
-	RsyncImage           string
-	Namespace            string
-	SetDefaults          bool
-	VerboseCopy          bool
-	SkipSourceValidation bool
+	SourceSCName               string
+	DestSCName                 string
+	RsyncImage                 string
+	Namespace                  string
+	SetDefaults                bool
+	VerboseCopy                bool
+	SkipSourceValidation       bool
+	SkipPVAccessModeValidation bool
+	PvcCopyTimeout             int
+	PodReadyTimeout            int
```

Comment:
```suggestion
	PvcCopyTimeout             time.Duration
	PodReadyTimeout            time.Duration
```

Why are we passing around unitless integers when there is literally a type for that

Reply:
My thought on this was to make easy on clients...just give me a time in seconds (int) and the implementation will convert it to the type(`time.Duration`) it needs.